package.json

```diff
@@ -50,7 +50,7 @@
     "@githubprimer/octicons-react": "^8.1.3",
     "@mdx-js/mdx": "^0.16.6",
     "@mdx-js/tag": "0.15.0",
-    "@primer/components": "^8.2.0-beta",
+    "@primer/components": "10.0.1",
```

Comment:
One thing, can you actually update this to 11.0.0? It will make the Heading the correct font weight :)